docs/maatregelen/index.md

```diff
@@ -1,71 +1,80 @@
- 
-## Standaarden
-Op nationaal, Europees en internationaal niveau wordt momenteel gewerkt aan de totstandkoming van standaarden.
-Deze standaarden worden opgesteld door gespecialiseerde organisaties (NEN, JTC21 en ISO) en ondergaan een uitgebreid proces.
-Dit proces zorgt uiteindelijk voor brede consensus voor de standaarden.
-Het naleven van deze standaarden door organisaties levert een vermoeden van conformiteit op, ofwel, door het naleven van standaarden is de kans groot dat organisaties voldoen aan de vereisten.
-De standaarden worden daarom een waardevolle bron voor het Algoritmekader.
-Waar mogelijk zullen de standaarden onderdeel worden van het Algoritmekader en tot uiting komen in de laag van 'maatregelen'.
-Er zal een verwijzing worden gemaakt naar de betreffende standaard.
-Hierover moeten nog nadere afspraken worden gemaakt met deze standaardisatieorganisaties.
-
-## Overzicht van maatregelen
-
-Onderstaand volgt een overzicht van maatregelen per fase van de levenscyclus. 
-
-??? simple "Probleemanalyse"
-
-    <!-- list_maatregelen_1 levenscyclus/probleemanalyse -->
-
-??? simple "Ontwerp"
-
-    <!-- list_maatregelen_1 levenscyclus/ontwerp -->
-
-??? simple "Dataverkenning en datapreparatie"
-
-    <!-- list_maatregelen_1 levenscyclus/dataverkenning-en-datapreparatie -->
-
-??? simple "Ontwikkelen"
-
-    <!-- list_maatregelen_1 levenscyclus/ontwikkelen -->
-
-??? simple "Verificatie en validatie"
-
-    <!-- list_maatregelen_1 levenscyclus/verificatie-en-validatie -->
-    
-??? simple "Implementatie"
-
-    <!-- list_maatregelen_1 levenscyclus/implementatie -->
-    
-??? simple "Monitoring en beheer"
-
-    <!-- list_maatregelen_1 levenscyclus/monitoring-en-beheer -->
-    
-??? simple "Uitfaseren"
-
-    <!-- list_maatregelen_1 levenscyclus/uitfaseren -->
-
+# Maatregelen
+Overzicht van maatregelen voor verantwoord gebruik van algoritmes en AI-systemen. Het zijn adviezen om te voldoen aan de eisen voor overheden. Andere maatregelen zijn ook mogelijk.
+
+## Alle maatregelen zijn adviezen
+De maatregelen zijn niet verplicht. Het zijn adviezen uit:
+- Toetsingskader Algoritmes, Algemene Rekenkamer
+- Onderzoekskader algoritmes, Auditdienst Rijk
+- nationale en internationale standaarden (NEN, JTC21 en ISO)
+- onze werkgroepen
+
+### Voorbeeld
+> Onderzoek het ontwikkelde algoritme op onbewuste vooringenomenheid (discriminatie) door middel van een bias-analyse.
+
+Deze maatregel helpt om te voldoen aan de eis om niet te discrimineren. Maar deze maatregel is niet verplicht. Je organisatie mag ook eigen maatregelen nemen. Zolang je uiteindelijk maar voldoet aan de eis.
+
+> [!TIP]
+> Kies maatregelen die gebaseerd zijn op een standaard. Je product krijgt hierdoor een ‘vermoeden van conformiteit’. Dit betekent dat de kans groot is dat je organisatie voldoet aan de eisen.
```

Comment:
Dit geldt alleen voor de AI verordening denk ik. Het is nu nog een lastige tip, omdat de standaarden er nog niet zijn

Reply:
Ah, dit is dus een tip voor de toekomst? Dan kan 'ie nu weg dus?

Reply:
```suggestion

```